requirements-docs.txt

```diff
@@ -1,5 +1,10 @@
 docutils<0.17,>=0.15  # sphinx-book-theme 0.2.0 has requirement docutils<0.17,>=0.15
 Sphinx<5,>=3  # sphinx-book-theme 0.3.3 has requirement sphinx<5,>=3
+sphinxcontrib-applehelp==1.0.4  # newer versions require sphinx 5
+sphinxcontrib-devhelp==1.0.2   # newer versions require sphinx 5
+sphinxcontrib-htmlhelp==2.0.1  # newer versions require sphinx 5
+sphinxcontrib-qthelp==1.0.3  # newer versions require sphinx 5
+sphinxcontrib-serializinghtml==1.1.5  # newer versions require sphinx 5
```

Comment:
Do we really need all those `sphinxcontrib-*help`s?

Reply:
@stevepiercy Yes, they are dependencies of Sphinx. They released new versions yesterday which are not compatible with Sphinx<5, but no longer declare that in their package metadata.

Reply:
I mean that we don't build those types of outputs, so why add them at all? We use only HTML and linkcheck.

Reply:
@stevepiercy And I meant that as far as I can tell, we don't have a choice, because they are pulled in by sphinx.